media-proxy/tests/conn_rdma_real_ep_test.cc

```diff
@@ -139,8 +139,7 @@ class RdmaRealEndpointsTest : public ::testing::Test {
     // Set up RDMA Rx
     mcm_conn_param rx_request = {};
     rx_request.type = is_rx;
-    rx_request.local_addr = {.ip = "192.168.10.21", .port = "8002"};
-    rx_request.remote_addr = {.ip = "192.168.10.20", .port = "8003"};
+    rx_request.local_addr = {.ip = "192.168.1.22", .port = "8002"};
```

Comment:
Avoid using `8002` here as it interferes with the legacy TCP listener socket port.

Reply:
Fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/293/commits/907a4e3917a16fa35b4ebde105a15f179531b040